web/src/components/LandingPageActionTile.tsx

```diff
@@ -33,15 +36,18 @@ export const LandingPageActionTile = (props: Props): ReactElement => {
   const isMobile = useMediaQuery(MediaQueries.isMobile);
 
   return (
-    <button
-      className={`${props.isPrimary ? "landing-page-tile" : "landing-page-secondary-tile"} margin-1 ${
-        isMobile ? "tal" : ""
-      }`}
-      data-testid={props.dataTestId}
-      onClick={props.onClick}
-    >
-      <img src={props.imgPath} alt="" />
-      {setTileText(props, isMobile)}
-    </button>
+    <div className={props.className}>
+      <button
+        ref={props.reference}
+        className={`${props.isPrimary ? "landing-page-tile" : "landing-page-secondary-tile"} margin-1 ${
+          isMobile ? "tal" : ""
```

Comment:
I must have missed this in previous stories:
1) You added this "tal" class to the "flex.scss" file but it is unrelated to flexbox and should have been added to something relevant like "typography.scss" because it has to do with text
2) The couple css classes we have that are three-letter abbreviations are an old pattern we're moving away from. This should have been named something descriptive like "text-align-left"
3) Regardless, neither of those matter because there is in fact already a class named "text-align-left" in the typography.scss file, so that should have been used instead of creating something new

Reply:
also - your landing-page-tile class already has a css media query on it. Why is this styling in javascript at all instead of just added into the media query in the css file?